netsim/ansible/templates/normalize/eos.j2

```diff
@@ -0,0 +1,6 @@
+{% for intf in interfaces if intf.virtual_interface is not defined %}
+!
+interface {{ intf.ifname }}
+ no switchport
+ mac-address {{ '52dc.cafe.%02d%02d' % ( id,intf.ifindex % 100 ) }}
```

Comment:
This will break in topologies with more than 255 nodes, or when users use a custom node ID > 255

Reply:
The attribute limit is 150 (based on the default management network settings), the hard-coded limit in nodes.py is 250 (because we use ID in things like MAC addresses). Anyway, the formatting string needs to be changed to hex, or anything over 99 will break things.